LiteCore/Support/TestsCommon.cc

```diff
@@ -81,7 +81,8 @@ void InitTestLogging() {
             C4Error error;
             if (!c4log_writeToBinaryFile({kC4LogVerbose, slice(path), 16*1024, 1, false},
                                          &error)) {
-                C4WarnError("TestsCommon: Can't log to binary file");
+                C4WarnError("TestsCommon: Can't log to binary file, %.*s",
```

Comment:
I think we don't need "TestsCommon:".

Reply:
Okay, I'll take it out, but it is not added by this PR.